docs/api/api.yml

```
@@ -0,0 +1,275 @@
+openapi: 3.0.1
+
+servers:
+  - description: Debug environment
+    url: http://localhost:<TODO:portnumber>/
```

```suggestion
    url: http://localhost:20210/
```

This should be in the config already. Please check.